HelpSource/Classes/SystemSynthDefs.schelp

```diff
@@ -0,0 +1,58 @@
+class:: SystemSynthDefs
+categories:: Server
+summary:: SynthDefs assumed to exist on the server
+related:: Classes/Server, Classes/SynthDef
+
+description::
+
+After it has booted, this collection of link::Classes/SynthDef::s is sent to the server. The SynthDefs are used for purposes such as routing audio and writing to busses, see link::Classes/Event:: and link::Classes/NodeProxy::. The SynthDefs are instantiated automatically at link::Classes/StartUp::. For non-realtime-synthesis, they have to be added explicitly, see link::Classes/Score#-addSystemSynthDefs::.
+
+When playing functions anonymously (link::Classes/Function#-play::), the temporary SynthDef names are also generated by this class.
+
+
+classmethods::
+
+subsection::System SynthDefs
+
+method::numChannels
+
+The SystemSynthDefs class generates SynthDefs with different number of channels that are chosen according to need. This class variable specifies the maximum number of channels.
+
+
+method::synthDefs
+
+The list of non-temporary synthDefs generated.
+
+code::
+SystemSynthDefs.synthDefs.do { |x| x.name.postln }; // post the names of SynthDefs
+::
+
+
+subsection::Temporary SynthDef names
+
+method::generateTempName
+
+Return a new temporary name
+
+code::
+SystemSynthDefs.generateTempName
+::
+
+method::tempNamePrefix
+
+A prefix name that can be used to distinguish Synths from other names to avoid collision.
+
+code::
+a = { SinOsc.ar * 0.01 }.play;
+a.defName; // has the prefix 'temp__', followed by a number
+::
+
+method:;maxTempDefNames
+
+A maximum count for the numbers to add to the tempNamePrefix
+
+
+
+
+
+
```

Comment:
Some excessive newlines - not a problem in general, but I think it would be cleaner to not have them.